sdk/cosmos/azure_data_cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.25.0 (Unreleased)
 
 ### Features Added
+* Added 'if_match_etag' to ItemOptions and necessary functions ([#2705](https://github.com/Azure/azure-sdk-for-rust/pull/2705))
```

Comment:
necessary functions? My suggestion is to either remove the word `necessary functions` or elaborate on that :)